### PR TITLE
DEV: various refinements to reactions-list-emoji

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

**Comment by jjaffeux** [comment]
@Ahmedgagan any comment on this please ?

File: assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js.es6 [hunk 1]
```diff
@@ -26,44 +28,10 @@ export default createWidget("discourse-reactions-list-emoji", {
...
-          name: "preventOverflow",
-          options: {
-            padding: 5
-          }
-        }
-      ]
-    });
+  didRenderWidget() {
+    if (!this[POPPER_NAME]) {
+      this._setupPopper(POPPER_NAME, ".user-list");
```
Comment (Ahmedgagan) [inline assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js.es6:33]: Can we not set up this popper when in mobileView? as we won't be using it in mobileview

**Comment by Ahmedgagan** [comment]
> @Ahmedgagan any comment on this please ?

I think it is perfect, just a small change which I mentioned. rest everything's perfect.

File: assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js.es6 (hunk 1 above)
Comment (jjaffeux) [reply assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js.es6:33]: yes indeed good call :+1:

**Pull request merged**